src/environmentd/src/bin/environmentd/main.rs

```diff
@@ -304,6 +304,11 @@ pub struct Args {
     )]
     orchestrator_process_data_directory: PathBuf,
 
+    /// The init container to use for computed and storaged when using the
+    /// kubernetes orchestrator.
+    #[clap(long)]
+    k8s_init_container_image: Option<String>,
```

Comment:
As we allow for different storaged and computed images to be specified, should we similarly break this up into `computed_init_container_image` and `storaged_init_container_image`?